package/Dockerfile

```diff
@@ -1,3 +1,12 @@
-FROM ubuntu:18.04
-COPY kontainer-engine-driver-lke /usr/bin/
-CMD ["kontainer-engine-driver-lke"]
+FROM golang:1.21 AS builder
+
+RUN apt-get install -y make
+
+COPY . /driver
+
+RUN cd /driver && make build
+
+FROM alpine:3.18
+
+COPY --from=builder /driver/kontainer-engine-driver-lke /usr/bin
+ENTRYPOINT ["kontainer-engine-driver-lke"]
```

Comment:
Because this Dockerfile is no longer published to a registry, it should _only_ be used for development purposes.